assets/js/archive-status.js

```diff
@@ -0,0 +1,63 @@
+// Load data attributes
+let script = document.getElementById("archive-status-script");
+let CSRF_TOKEN = document.getElementById("data-csrf-token");
+let DOWNLOAD_BAR_ID = script.getAttribute("data-download-bar-id");
+let DOWNLOAD_URL = script.getAttribute("data-download-url");
+let CREATE_ARCHIVE_URL = script.getAttribute("data-create-archive-url");
+let ARCHIVE_AVAILABILITY_URL = script.getAttribute("data-archive-availability-url");
+
+let spinner = $(".spinner");
```

Comment:
jQuery wraps the spinner in a data structure and subsequently allows for simple operations such as `.hide()` or `.show()`, the latter being used in the script instead of
```javascript
document.getElementById('spinner').style.display = 'inline-block'; 
```

Reply:
Nonetheless, I will add an `id` value for this element, since it's a singleton.